os/board/rtl8730e/src/component/soc/amebad2/fwlib/ram_common/ameba_i2c.c

```diff
@@ -715,8 +715,15 @@ u32 I2C_MasterRead(I2C_TypeDef *I2Cx, u8 *pBuf, u32 len)
 		while ((I2C_CheckFlagState(I2Cx, I2C_BIT_RFNE)) == 0) {
 			if (I2C_GetRawINT(I2Cx) & I2C_BIT_TX_ABRT) {
 				I2C_ClearAllINT(I2Cx);
-				return cnt;
+				return -1;
```

Comment:
what is the error?
Let's seperate error type.
If this error is needed try again, Please return -EAGAIN or 0

Reply:
I have changed all to -EAGAIN